queries/java/highlights.scm

```diff
@@ -59,32 +73,35 @@
   scope: (identifier) @type)
  (#match? @type "^[A-Z]"))
 
-(constructor_declaration
-  name: (identifier) @type)
-
-(type_identifier) @type
-(boolean_type) @type.builtin
-(integral_type) @type.builtin
-(floating_point_type) @type.builtin
-(floating_point_type) @type.builtin
-(void_type) @type.builtin
+[
+(boolean_type)
+(integral_type)
+(floating_point_type)
+(floating_point_type)
+(void_type)
+] @type.builtin
 
 ; Variables
 
 ((identifier) @constant
  (#match? @constant "^_*[A-Z][A-Z\d_]+"))
 
 (identifier) @variable
```

Comment:
This is not highlighted.

Reply:
Non we don't this highlighting is not useful, could you remove it?

Reply:
ok

Reply:
No modification here ?

Reply:
idk it didn't save properly

Reply:
i deleted those

Reply:
It isn't there, i think gh messed up. Please see https://raw.githubusercontent.com/PitcherTear22/nvim-treesitter/master/queries/java/highlights.scm

Reply:
Find says phrase not found https://imgur.com/O9N6DIM

Reply:
I was referring to `@variable`

Reply:
`identifier` or `this`?
`this` has been commented out as TODO

Reply:
Just delete `(identifier) @variable`. We have no highlight for `@variable`. This is what @vigoux was trying to say.